AmahiAnywhere/Podfile.lock

```diff
@@ -17,7 +17,7 @@ PODS:
   - Lightbox (2.3.0):
     - Imaginary (~> 4.0)
   - MBProgressHUD (1.2.0)
-  - MobileVLCKit (3.3.13)
+  - MobileVLCKit (3.3.12)
```

Comment:
Please pay attention to this. It's kind of obvious (I hope) that we should not go back on something big like this.